.github/workflows/ansible-test-plugins.yml

```
@@ -16,34 +16,31 @@ on:
 
 jobs:
   sanity:
-    name: "Sanity (Python: ${{ matrix.python }}, Ansible: ${{ matrix.ansible }})"
+    name: "Sanity (Ansible: ${{ matrix.ansible }})"
     runs-on: ubuntu-latest
     strategy:
       matrix:
         ansible:
           - stable-2.9
           - stable-2.10
           - devel
-        python:
-          - 2.7
-          - 3.6
-          - 3.7
-          - 3.8
-    steps:
 
+    steps:
       - name: Check out code
         uses: actions/checkout@v2
         with:
           path: ansible_collections/gluster/gluster
 
-      - name: Set up Python ${{ matrix.python }}
+      # Just run once for the target python version
+      - name: Set up Python ${{ python-version }}
         uses: actions/setup-python@v2
         with:
-          python-version: ${{ matrix.python }}
+          # 3.8 is blocked by ansible/ansible#70155
+          python-version: 3.7
```

```suggestion
          python-version: 3.8
```
This only matters if you don't use `--docker` in ansible-test.